bedrock/cms/tests/test_models.py

```diff
@@ -0,0 +1,69 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from unittest import mock
+
+import pytest
+
+from bedrock.cms.models import AbstractBedrockCMSPage, SimpleRichTextPage
+from bedrock.cms.tests.factories import StructuralPageFactory
+
+pytestmark = [
+    pytest.mark.django_db,
+]
```

Comment:
TIL: You can apply marks at a module level. Looks like it works as a single mark and as a list.

Reply:
Yeah that's right. I have a habit of marking things up as a list to make future diffs smaller and show that multiple marks are poss